### PR TITLE
Mileston2 - Calculator 

### PR DESCRIPTION
**Pull request by gasb150**
For this milestone was necessary: 
Create a components directory in the src folder.
with four files (components) in there: App.js, Display, ButtonPanel and Button.js
Implement the App component.

1. The App renders the  Display and ButtonPanel component.
2. The Display component accepts the result of the calculation as an incoming prop.
 - The result is a String and the default value of the "result" prop is 0.
3. The Button component
 - It accepts a button name as a prop.
 - The button name prop is a String.
4. ButtonPanel component renders the calculator panel with all the buttons in groups.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-
-const App = () => (
-  <div>
-    <h1>Calculator App</h1>
-  </div>
-);
+import App from './components/App';
 
 ReactDOM.render(
   <App />,
```
Comment (teekaytech) [inline src/index.js:6]: The `src/index.js` should:

- [ ] add example books to initial state object <a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png">screenshot from pathwright</a>
- [ ] import { Provider } from react-redux

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const result = '0';
+const App = () => (
+  <>
+    <Display result={result} />
+    <ButtonPanel />
+
+  </>
+);
+
+export default App;
```
Comment (teekaytech) [inline src/components/App.js:14]: The `App` component should display the `BooksList` and `BookForm` components

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @gasb150 :wave:

__I must commend your efforts so far in this project. Here are the major highlights for me:__

- GitHub Action checks are running/passing successfully :100: 
- The PR title and summary are descriptive :ok_hand: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] The `Book`, `BookList` and `BookForm` components are completely missing. Kindly go through the milestone requirements all over again and ensure that you implement all the necessary components with their features.
- [ ] You're not expected to tamper with the linters configuration files. I can see some activities around the files: you should use the default configuration setup.
- [ ] The Redux Actions files (`src/actions/index.js`, `src/actions/books.js`) are completely missing. 
- [ ] Check my inline comments below and fix them as appropriate. 

**[ OPTIONAL CHANGES ]**
N/A


* * *
**Please make the required changes and submit another review request when you are done.**

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: src/components/App.js (hunk 2 above)
Comment (gasb150) [reply src/components/App.js:14]: Hello @teekaytech, I think you are reviewing a wrong project, I'm asking for Calculator project.

Comment (teekaytech) [reply src/components/App.js:14]: Oh yeah! 
Sorry about that @gasb150, I'm gonna re-review now.

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @gasb150 :tada: :clinking_glasses: 

**This milestone has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**


Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**